docs/getting-started.md

```diff
@@ -0,0 +1,116 @@
+## getting started
+
+You'll need **python 3.6+** and **Google Chrome**.
+The web UI is tested on OSX and Windows using Chrome, and the python CLI is tested on OSX and Ubuntu (via WSL/Windows). It should work on other platforms, but if you run into trouble let us know.
+
+To install run
+
+```
+pip install cellxgene
+```
+
+To start exploring a dataset call
+
+```
+cellxgene launch dataset.h5ad --open
+```
+
+If you want an example dataset download [this file](https://github.com/chanzuckerberg/cellxgene/raw/master/example-dataset/pbmc3k.h5ad) and then call
+
+```
+cellxgene launch pbmc3k.h5ad --open
+```
+
+You should see your web browser open with the following
```

Comment:
"If you are on MacOs, you should see your web browser ..."

Reply:
This should work in Ubuntu, too (it does for jupyter, which also uses `webbrowser`). Is it just WSL where it breaks?

Reply:
Yes, likely just WSL due to the split-brain model...